packages/pkg-tests/.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Also not sure about this one. Can we please re-use the lint rules from the main folder? I don't think this test framework/suite should adhere to different lint rules than others.